core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -152,13 +156,19 @@ class KafkaApis(val requestChannel: RequestChannel,
     // ensureTopicExists is only for client facing requests
     // We can't have the ensureTopicExists check here since the controller sends it as an advisory to all brokers so they
     // stop serving data to clients for the topic being deleted
-    val stopReplicaRequest = request.requestObj.asInstanceOf[StopReplicaRequest]
+    val stopReplicaRequest = request.body.asInstanceOf[StopReplicaRequest]
 
-    authorizeClusterAction(request)
```

Comment:
@ijuma I had started moving away from that method because authorization, to me, is not an exceptional case the should be caught but an explicit case that should be handled. It was discussed some on previous pull requests. I am guessing @dajac followed the pattern in my most recent patches.